.github/workflows/tests.yml

```diff
@@ -0,0 +1,62 @@
+name: Test generators
+on: [pull_request, push]
+
+env:
+  IMAGE_NAME: generators
+  IMAGE_PATH: /tmp/generators.tar
+
+jobs:
+  build-docker:
```

Comment:
It was done this way before I've created this script. Then I left it because it saves a little bit of time, since building docker and building test library can be done in parallel. The gain is ~1.5 minute